x-pack/plugin/data-frame/src/main/java/org/elasticsearch/xpack/dataframe/transforms/DataFrameIndexer.java

```diff
@@ -73,6 +96,12 @@ protected void onStart(long now, ActionListener<Void> listener) {
         }
     }
 
+    @Override
+    protected void onFinish(ActionListener<Void> listener) {
+        // reset the page size, so we do not memorize a low page size forever, the pagesize will be re-calculated on start
+        pageSize = 0;
```

Comment:
Does this ever get called? I don't see `super.onFinish` being called anywhere other than the test.

Reply:
;-) forgot to save and push